library/patient.inc

```diff
@@ -812,6 +814,15 @@ function getPatientPID($args)
 
     return $returnval;
 }
+function getPatientDoBonly($pid)
+{
+    if (empty($pid)) {
+        return "";
+    }
+
+    $sql = sqlQuery("SELECT DOB FROM patient_data WHERE pid = ?", array($pid));
```

Comment:
This _could_ be cleaned up a little. This is getting *really* nit picky but see:

```php
$sql = "SELECT DOB FROM patient_data WHERE pid = ?";
$query = sqlQuery($sql, [$pid]);
```

Yes it's an extra line of seemingly redundant code, but it's cleaner to follow - especially in the really long SQL statements. Again, I'm getting into hair-splitting territory here

Reply:
Yes, this is picky but good because I paused when writing this part. There where three ways it could have been written based on what I know that is in the code base but I when with what I see the most of. This was my second choice. I like it better.

Reply:
Just because you see it everywhere in the code doesn't mean it's what we have to keep doing :)

The codebase is fraught with less than great practices - it's a slooooowww fix that starts with new code being introduced in a way that helps others see what could be